types/da.go

```diff
@@ -22,10 +22,11 @@ func SubmitWithHelpers(
 	logger zerolog.Logger,
 	data [][]byte,
 	gasPrice float64,
-	namespace []byte, // New namespace parameter
+	namespace string,
 	options []byte,
 ) coreda.ResultSubmit { // Return core ResultSubmit type
-	ids, err := da.SubmitWithOptions(ctx, data, gasPrice, namespace, options)
+	ns := coreda.NamespaceFromString(namespace)
```

Comment:
Why call this every time instead of once on start up and pass it down?

Reply:
https://github.com/evstack/ev-node/pull/2683/commits/044cf0cf443229b25cb7a846ade911cf0b0aac7f